### PR TITLE
Added table of contents.

### PR DESCRIPTION
**Pull request by eeswr**

### CODE REVIEW

File: src/pages/api/index.md [hunk 1]
```diff
@@ -7,6 +7,23 @@ The Seneca API is pretty tiny. We try hard to keep any non-essential functionali
 core. Instead, we push it out to plugins. The core API is documented below. If you have
 any further questions, get in touch. We love to talk!
 
+## Table of Contents
+- [Seneca ([options]): this](#seneca-options-this)
```
Comment (mcdonnelldean) [inline src/pages/api/index.md]: - remove space between Seneca and (
- remove `:this`

File: src/pages/api/index.md [hunk 2]
```diff
@@ -7,6 +7,23 @@ The Seneca API is pretty tiny. We try hard to keep any non-essential functionali
...
+- [close([done]) : void](#close-done-void)
```
Comment (mcdonnelldean) [inline src/pages/api/index.md]: remove `:this`

File: src/pages/api/index.md [hunk 3]
```diff
@@ -7,6 +7,23 @@ The Seneca API is pretty tiny. We try hard to keep any non-essential functionali
...
+- [act (msg [,payload][,reply]):this](#act-msg-payload-reply)
```
Comment (mcdonnelldean) [inline src/pages/api/index.md]: - remove space between act and (
- remove `:this`

File: src/pages/api/index.md [hunk 4]
```diff
@@ -7,6 +7,23 @@ The Seneca API is pretty tiny. We try hard to keep any non-essential functionali
...
+- [add(pattern [,spec], handler):this](#add-pattern-spec-handler-this)
```
Comment (mcdonnelldean) [inline src/pages/api/index.md]: - remove `this`

File: src/pages/api/index.md [hunk 5]
```diff
@@ -7,6 +7,23 @@ The Seneca API is pretty tiny. We try hard to keep any non-essential functionali
...
+- [has(pattern):boolean](#has-pattern-boolean)
```
Comment (mcdonnelldean) [inline src/pages/api/index.md]: - remove `:boolean`

File: src/pages/api/index.md [hunk 6]
```diff
@@ -7,6 +7,23 @@ The Seneca API is pretty tiny. We try hard to keep any non-essential functionali
...
+- [sub(pattern, handler)](#sub-pattern-handler)
+- [listen(options)](#listen-options-)
+- [client(options)](#client-options-)
+- [use(plugin[,options]):this](#use-plugin-options-this)
```
Comment (mcdonnelldean) [inline src/pages/api/index.md]: - remove `:this`

File: src/pages/contribute/adding-a-tutorial.md [hunk 7]
```diff
@@ -7,7 +7,7 @@ The process of adding a tutorial or guide for seneca is very easy. All of our do
 in markdown with the ability to mix in HTML as needed too.
 
 Assuming you have fork of this site, your tutorial or guide should live in the
-[/src/pages/tutorials][] folder.
+[hello][] folder.
```
Comment (mcdonnelldean) [inline src/pages/contribute/adding-a-tutorial.md]: ??

Comment (eeswr) [inline src/pages/contribute/adding-a-tutorial.md]: Sorry this is left over from getting used to markdown links.

**Comment by mcdonnelldean** [comment]
Good stuff @eeswr!

Made a couple of notes. I'm not sure what the hello change is for. Ping me on slack about that one.

File: src/pages/contribute/adding-a-tutorial.md [hunk 8]
```diff
@@ -43,4 +43,4 @@ your PR is accepted your changes will go live. You do not need to take any speci
 
 [localhost:4000]: http://localhost:4000/
 [/src/pages/tutorials/index.md]: https://github.com/senecajs/senecajs.org/blob/master/src/pages/tutorials/index.md
-[/src/pages/tutorials]: https://github.com/senecajs/senecajs.org/blob/master/src/pages/tutorials
+[hello]: https://github.com/senecajs/senecajs.org/blob/master/src/pages/tutorials
```
Comment (mcdonnelldean) [inline src/pages/contribute/adding-a-tutorial.md]: @eeswr this needs to be reverted too.

**Pull request merged**